frontend/src/store/modules/visitors/actions.js

```diff
@@ -16,4 +33,30 @@ export default {
             context.commit(RESET_BUTTON_FETCHING, data.button);
         }
     },
+    [FETCH_TABLE_DATA]: (context, data) => {
+        if (!data.value) {
+            return;
+        }
+        context.commit(SET_TABLE_DATA_FETCHING);
+        context.commit(GET_ACTIVE_BUTTON)
```

Comment:
you will not get data wit this approach. Take active button from context.state

Reply:
Fixed